examples/subprocess-output/subprocess-output.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const React = require('react');
+const {render, Text, Box} = require('../../build');
+const childProcess = require('child_process');
+
+const SubprocessOutput = () => {
+	const [output, setOutput] = React.useState('');
+
+	React.useEffect(() => {
+		const subProcess = childProcess.spawn('ping', ['8.8.8.8']).stdout;
+
+		subProcess.on('data', newOutput => {
+			setOutput(newOutput.toString('ascii'));
+		});
+	}, [setOutput]);
+
+	return (
+		<Box flexDirection="column">
+			<Text>My ping output:</Text>
+			<Text>{output}</Text>
```

Comment:
To give it a little more breathing room. It's also probably a good idea to limit the number of lines you can print, to ensure it doesn't fill up the entire terminal window.

```suggestion
			<Box marginTop={1}>
			    <Text>{output}</Text>
			 </Box>
```